backend/routes/users.js

```diff
@@ -168,4 +168,78 @@ router.get("/getStudentData", async (req, res) => {
   }
 });
 
+/**
+ *
+ * @api {post} /assignStudentMajor
+ * @apiDescription assign or update a Major curriculum to a student
+ * @apiParam (body) {Integer} studentID, {string} major
+ * @apiExample {curl} Example usage:
+ *	curl -X POST \
+ *  -H 'Content-Type: application/json' \
+ *  -d '{"studentID": 260678788, "major": "Electrical Engineering"}' \
+ *  http://localhost:3001/courses/assignStudentMajor
+ *
+ * @returns True on success
+ *          invalid student ID
+ *          invalid curriculum name
+ *          student does not exist
+ *          curriculum does not exist
+ *          student already assigned major
+ *
+ * @author: Gareth Peters
+ *
+ */
+
+router.post("/assignStudentMajor", async (req, res) => {
+  const studentID = req.query.studentID;
+  const major = req.query.major;
+  try {
+    await users.assignStudentMajor(studentID, major);
+    res.status(200).send(true);
```

Comment:
@AlexLam1997  I think now we have to wrap the endpoint into `customResponse` right ?